travis-scripts/after_script.sh

```diff
@@ -16,7 +16,9 @@ test "$TRAVIS_PULL_REQUEST" = "false" && BRANCH_OR_PULL_REQUEST=$TRAVIS_BRANCH |
 filename=cfengine-$VERSION-$BRANCH_OR_PULL_REQUEST-$JOB_TYPE.artifacts.zip
 zip -q -r $filename artifacts/
 
-echo ===== uploading $filename to transfer.sh =====
-curl --upload-file $filename https://transfer.sh/$filename
+echo ===== uploading $filename to file.io =====
+curl -F "file=@$filename" https://file.io
+echo 'Note that file.io DELETES file from their servers after first download,'
+echo "so don't delete it from your machine if you still need it!"
```

Comment:
I suppose we only need to download the file once.? Or are multiple travis jobs using the same file?

Reply:
No, we never download these files. They are there only for the case if someone wants to check a travis failure - they can always download artifacts.

Reply:
OK, cool. I wasn't sure if this wasn't used by the other jobs downloading the packages produced by travis builds.

Reply:
+1. A comment about this would be nice.